From 00063c587b75ea5ff6881419e184eb410f68d0a7 Mon Sep 17 00:00:00 2001
From: Wei Sun <wei.w.sun@intel.com>
Date: Fri, 8 May 2015 14:27:35 +0800
Subject: [PATCH 388/462] ov5693: add 480p sensor setting for preview

add 480p sensor setting for preview to avoid dropping frame
when run cts android.hardware.cts.CameraGLTest#testCameraTo
SurfaceTextureMetadata

Change-Id: Ied2e5efbcb803349f33853524964c47bda00547f
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-35416
Signed-off-by: Wei Sun <wei.w.sun@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/55113
Reviewed-by: Zaikuo Wang <zaikuo.wang@intel.com>
Build: IMIN (SC)
Reviewed-by: Fang Wang <fang.c.wang@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Reviewed-by: Yunliang Ding <yunliang.ding@intel.com>
Approver: Tian Shu Qiu <tian.shu.qiu@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 .../camera/drivers/media/i2c/ov5693-ecs/ov5693.h           | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
index 7a25945..20fb15d 100755
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov5693-ecs/ov5693.h
@@ -1057,6 +1057,20 @@ static struct ov5693_reg const ov5693_2576x1936_30fps[] = {
 
 struct ov5693_resolution ov5693_res_preview[] = {
 	{
+		.desc = "ov5693_736x496_30fps",
+		.width = 736,
+		.height = 496,
+		.pix_clk_freq = 160,
+		.fps = 30,
+		.used = 0,
+		.pixels_per_line = 2688,
+		.lines_per_frame = 1984,
+		.bin_factor_x = 1,
+		.bin_factor_y = 1,
+		.bin_mode = 0,
+		.regs = ov5693_736x496_30fps,
+	},
+	{
 		.desc = "ov5693_1616x1216_30fps",
 		.width = 1616,
 		.height = 1216,
-- 
1.9.1

